src/components/swap/components/transaction-review/index.tsx

```diff
@@ -7,31 +8,32 @@ import {
   ModalContent,
   ModalHeader,
   ModalOverlay,
-  Text,
 } from '@chakra-ui/react'
 
-import { useColorStyles } from '@/lib/styles/colors'
+import { colors } from '@/lib/styles/colors'
 
 import { Review } from './components/review'
 import { SubmissionResult } from './components/submission-result'
 import { TransactionReview } from './types'
 
+import './styles.css'
+
 enum TransactionReviewModalState {
   failed,
   submit,
   success,
 }
 
 type TransactionReviewModalProps = {
+  isDarkMode?: boolean
```

Comment:
For now the easiest solution I saw, as chakra won't pick up `dark` for the modal component if set on parent components (that open the modal).